packages/components/note/src/Note.tsx

```diff
@@ -73,11 +77,19 @@ export const Note = React.forwardRef<HTMLElement, ExpandProps<NoteProps>>(
       onClose,
       testId = 'cf-ui-note',
       title,
+      startIcon,
       ...otherProps
     } = props;
 
     const styles = getNoteStyles();
 
+    const iconContent = (icon: React.ReactElement) =>
+      React.cloneElement(icon, {
+        size: icon.props.size ?? `${title ? 'medium' : 'small'}`,
+        variant:
```

Comment:
I'm wondering if we should lock the icon variant to the note variant, the same way we do with the button component: https://github.com/contentful/forma-36/blob/fafa03f7c2f2d4425e906e4c9a227a61d3816641/packages/components/button/src/Button/Button.tsx#L54-L81 – my gut feeling says yes, we can always allow more customisation later on if it turns out to be needed.

Reply:
Good call, let's it keep closed for now.